client/signup/config/flows-pure.js

```diff
@@ -113,7 +114,9 @@ export function generateFlows( {
 			steps: isEnabled( 'signup/professional-email-step' )
 				? [ 'user', 'domains', 'emails', 'plans' ]
 				: [ 'user', 'domains', 'plans' ],
-			destination: getSignupDestination,
+			destination: isEnabled( 'signup/setup-site-after-checkout' )
+				? getSetupSiteDestination
+				: getSignupDestination,
 			description: 'Abridged version of the onboarding flow. Read more in https://wp.me/pau2Xa-Vs.',
 			lastModified: '2020-12-10',
```

Comment:
It appears that this should be updated when we change this method. it looks like you updated it in the other PR already https://github.com/Automattic/wp-calypso/pull/55883/files#diff-004d43d24ca95a3f3a3b10e34e557ebc0348147c8c806c7c311b7909385e0a3aR421 I'm actually not sure why the lastModified property would be necessary, my guess would be something to do with tracking but I can't find code for that.

Reply:
Honestly I think it's just for documentation. It's probably not even that important to update until users actually start using this flow. Then we'd probably like to be able to see from the git history when the flow was changed.